.github/workflows/non-release.yaml

```
@@ -21,7 +21,7 @@ permissions: {}
 jobs:
   build:
     if: ${{ github.event_name != 'pull_request_target' || (github.event_name == 'pull_request_target' && github.event.label.name == vars.DEFAULT_LABEL_OK_TO_TEST && vars.DEFAULT_LABEL_OK_TO_TEST != '') }}
-    uses: ./.github/workflows/build.yaml
+    uses: $/.github/workflows/build.yaml
```

actionlint (1.7.12)

[error] reusable workflow call "$/.github/workflows/build.yaml" at "uses" is not following the format "owner/repo/path/to/workflow.yml@ref" nor "./path/to/workflow.yml". see https://docs.github.com/en/actions/learn-github-actions/reusing-workflows for more details

(workflow-call)